dao/src/main/resources/io/syndesis/dao/deployment.json

```diff
@@ -1,1897 +1,2625 @@
 [
-  {
```

Comment:
It looks like that the formatting has been changed making the diff quite meaningless (and hence not possible to review here). Maybe a tab issue ?